index.html

```diff
@@ -0,0 +1,416 @@
+<!--                </p>-->
+<!--                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. -->
+<!--                    Esse, laudantium non rem natus maiores qui officia.-->
+<!--                    Magnam ad ducimus velit beatae fugit? Cumque accusantium -->
+<!--                    cum numquam praesentium doloremque recusandae repellat!</p>-->
+<!--                    </p>-->
+<!--        </div> -->
+    </section>
+
+<!--    <div>-->
+<!--            <h1>The aside element - Styled with CSS</h1>-->
+<!--            -->
+<!--            <p>My family and I visited The Epcot center this summer. The weather was nice, and Epcot was amazing! I had a great summer together with my family!</p>-->
+
+<!--            <aside>-->
+<!--                <div id="titlepicks">-->
+<!--                    <h3>Editors' Picks</h3>-->
+<!--                </div>-->
+<!--                <div class="articles-links">-->
+<!--                    <a href="https://www.nytimes.com/2020/11/24/us/Utah-monolith-red-rock-country.html?action=click&algo=top_conversion&block=editors_picks_recirc&fellback=true&imp_id=424276028&impression_id=27848650-2f39-11eb-998e-255129888bff&index=0&pgtype=Article&region=ccolumn&req_id=160481777&surface=home-featured">-->
+<!--                        <img src="assets/article1.jpg">-->
+<!--                        <h5>A Weird Monolith Is Found in the Utah Desert</h5>-->
+<!--                    </a>-->
+<!--                    <a href="https://www.nytimes.com/2020/11/24/books/review/the-last-american-aristocrat-henry-adams-david-s-brown.html?action=click&algo=top_conversion&block=editors_picks_recirc&fellback=true&imp_id=878903625&impression_id=27848651-2f39-11eb-998e-255129888bff&index=1&pgtype=Article&region=ccolumn&req_id=160481777&surface=home-featured">-->
+<!--                        <img src="assets/article2.jpg">-->
+<!--                        <h5> The Brilliant, Bitter, Unlikable Scion of an American Political Dynasty</h5>-->
+<!--                    </a>-->
+<!--                    <a href="https://www.nytimes.com/article/top-book-lists.html?action=click&algo=top_conversion&block=editors_picks_recirc&fellback=true&imp_id=877282866&impression_id=27848652-2f39-11eb-998e-255129888bff&index=2&pgtype=Article&region=ccolumn&req_id=160481777&surface=home-featured">-->
+<!--                        <img src="assets/article3.jpg">-->
+<!--                        <h5> The 10 best books Through time </h5>-->
+<!--                    </a>-->
+<!--                </div>-->
+<!--            </aside>-->
+<!--            <p>My family and I visited The Epcot center this summer. The weather was nice, and Epcot was amazing! I had a great summer together with my family!</p>-->
+<!--            <p>My family and I visited The Epcot center this summer. The weather was nice, and Epcot was amazing! I had a great summer together with my family!</p>-->
+<!--    </div>-->
```

Comment:
Leaving commented codes in your project is highly discouraged, please remove them